recipe/scripts/deactivate.sh

```diff
@@ -1,5 +1,5 @@
 export GSETTINGS_SCHEMA_DIR=$GSETTINGS_SCHEMA_DIR_CONDA_BACKUP
-unset GSETTINGS_SCHEMA_DIR_CONDA_BACKUP
+unset GSETTINGS_SCHEMA_DIR_CONDA_BACKUP || true
```

Comment:
Shouldn't we also use the same procedure here as in the line below?

Reply:
It's protected by the -z test which should have the same effect. I like my approach since it's terser but wanted to keep the diff minimal. Thanks for reviewing!